pkg/virt-handler/vm.go

```diff
@@ -1465,10 +1465,6 @@ func (d *VirtualMachineController) calculateLiveMigrationCondition(vmi *v1.Virtu
 		return newNonMigratableCondition(tscRequirement.Reason, v1.VirtualMachineInstanceReasonNoTSCFrequencyMigratable), isBlockMigration
 	}
 
-	if vmi.IsCPUDedicated() && vmi.Spec.Domain.CPU.IsolateEmulatorThread {
```

Comment:
Do we block RT?

Reply:
@xpivarc no

Reply:
This might be dangerous, for example, right now an eviction can trigger a migration of RT VM without the user`s intention.

Reply:
Well, that's only if you put `EvictionStrategy: LiveMigrate` in the VM spec, so all good I'd say.